### PR TITLE
2021-08-19-Liquid Global

### PR DESCRIPTION
**Pull request by Rufus602**

### CODE REVIEW

**Comment by evgenydmitriev** [comment]
Before we review the content, please fix the file name and avoid using backtick quotes (`) - they [mean something different](https://golem.ph.utexas.edu/~distler/maruku/markdown_syntax.html#:~:text=To%20indicate%20a%20span%20of%20code%2C%20wrap%20it%20with%20backtick%20quotes%20(%60)) in markdown.

**Comment by Rufus602** [comment]
> False. The search results show that Liquid Global did provide an estimate for the loss. Multiple sources report that the hackers stole about $97 million worth of cryptocurrencies from Liquid Global. Elliptic, a blockchain analytics company, also confirmed that about $97 million in cryptocurrencies were stolen. Liquid Global stated that it was tracing the movement of the stolen cryptocurrencies and working with other exchanges to freeze and recover the assets.

Liquid Global did not provided official looses since it is regulated Japans's Financial Services Agency.

**Comment by Rufus602** [comment]
> There is no evidence in the search results to support or refute the claim. The search results only mention that the hackers stole various cryptocurrencies from the exchange, but do not provide any details on the wallets used to execute the attack.

This is link to the official blog of liquid global, where it is claimed that thief used  ERC20
https://blog.liquid.com/warm-wallet-incident

**Comment by Rufus602** [comment]
> There is no evidence in the search results to support or refute the claim that FTX lent $120 million to Liquid Global. The search results only show that Liquid Global suffered a hack and lost $97 million worth of cryptocurrencies.

Multiple articles says opposite, it is one of them: https://www.coindesk.com/business/2021/08/25/recently-hacked-exchange-liquid-global-secures-120m-loan-from-ftx/

**Comment by Rufus602** [comment]
> The claim is false. According to multiple sources, including CoinDesk and CNBC, Liquid Global was hacked on August 19, 2021, resulting in the theft of $97 million worth of cryptocurrency. As a result, the exchange suspended deposits and withdrawals. There is no evidence in the search results to suggest that Liquid Global has fully resumed deposits and withdrawals for all cryptocurrencies.

This link provides evidence:
https://blog.liquid.com/warm-wallet-incident

**Comment by Rufus602** [comment]
> The claim is false. The search results show that Liquid Global suffered a major hack in August 2021 and lost $97 million worth of cryptocurrencies. However, on August 25, 2021, it was reported that FTX would lend $120 million to Liquid Global. There is no evidence to support the claim that the loan was closed on February 2, 2022. Therefore, the claim cannot be verified and the verdict is false.

https://blog.liquid.com/liquid-ftx-dept-financing

**Comment by Rufus602** [comment]
> /qualitycheck

please re-review article since bot is wrong

**Comment by Rufus602** [comment]
> Before we review the content, please fix the file name and avoid using backtick quotes (`) - they [mean something different](https://golem.ph.utexas.edu/~distler/maruku/markdown_syntax.html#:~:text=To%20indicate%20a%20span%20of%20code%2C%20wrap%20it%20with%20backtick%20quotes%20(%60)) in markdown.

Corrected in second commit

**Review by jhirschkorn — CHANGES_REQUESTED** [review]

File: content/attacks/posts/Liquid Global [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+date: 2021-08-19
+custodians: Liquid Global
+categories: Custodian
+title: Liquid Global suffered the hack loosing about $97 million worth of digital coins
```
[the remark this answers is not in the document]
Comment (jhirschkorn) [reply content/attacks/posts/Liquid Global:5]: double quotes -->"

**Review by jhirschkorn — CHANGES_REQUESTED** [review]

**Comment by jhirschkorn** [comment]
please add links to the sources you cite here in the article

**Comment by jhirschkorn** [comment]
status?

**Comment by JediFaust** [comment]
The last activity was 3 weeks ago, closing

**Pull request closed without merging**